code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -112,6 +112,10 @@
 		beaker = null
 		updateUsrDialog()
 
+/obj/machinery/atmospherics/components/unary/cryo_cell/update_icon()
+	update_icon_state()
+	return ..()
```

Comment:
This doesnt seem to be right, update_icon is already supposed to call update_icon_state you should rather fix update_icon_state not being called when applicable

Reply:
Alright, I'll move it to the atmospherics/components update_icon. It was missing that call, so I assumed it was doing so for some good reason.